node/pkg/p2p/p2p.go

```diff
@@ -46,6 +46,9 @@ const DefaultPort = 8999
 const P2P_VALIDATE_QUEUE_SIZE = 1024
 const P2P_SUBSCRIPTION_BUFFER_SIZE = 1024
 
+// TESTNET_BOOTSTRAP_DHI configures how many nodes may connect to the testnet bootstrap node. This number should not exceed HighWaterMark.
```

Comment:
Should we `assert(TESTNET_BOOTSTRAP_DHI < HighWaterMark)` if we want to enforce that?

Reply:
we could, but we don't usually do that elsewhere in the guardian. I'd love it if there was some way to do that at compile time, but I don't think there is.